schemas/yazi.json

```diff
@@ -141,6 +134,9 @@
 				"fetchers": { "$ref": "#/$defs/fetchers" },
 				"prepend_fetchers": { "$ref": "#/$defs/fetchers" },
 				"append_fetchers": { "$ref": "#/$defs/fetchers" },
+				"spotters": { "$ref": "#/$defs/spotters" },
+				"prepend_spotters": { "$ref": "#/$defs/spotters" },
+				"append_spotters": { "$ref": "#/$defs/spotters" },
```

Comment:
There is no definition for `#/$defs/spotters`. See the `$defs` object at the bottom. Should just have `name`, `mime`, and `run`, so taking from the `previewers` def:

```json
		"spotters": {
			"type": "array",
			"items": {
				"type": "object",
				"properties": {
					"name": { "type": "string" },
					"mime": { "type": "string" },
					"run": { "type": "string" }
				},
				"oneOf": [
					{
						"required": ["mime", "run"]
					},
					{
						"required": ["name", "run"]
					}
				]
			}
		},
```